tutorials/wordle.md

```diff
@@ -658,70 +909,140 @@ tx: null
 txhash: F159E11116EC9505FC2C0D97E605357FEC0F3DAE06B57BFB17EA6A548905043E
 ```
 
+Let's hold onto the txhash as we will need it to query the transaction.
+
+```bash
+TX_HASH=F159E11116EC9505FC2C0D97E605357FEC0F3DAE06B57BFB17EA6A548905043E
+```
+
 Note, this does not mean the transaction was included in the block yet.
 Let's query the transaction hash to check whether it has been included in
 the block yet or if there are any errors.
 
 <!-- markdownlint-disable MD013 -->
 ```bash
-rollkit query tx --type=hash F159E11116EC9505FC2C0D97E605357FEC0F3DAE06B57BFB17EA6A548905043E --chain-id wordle --output json | jq -r '.raw_log'
+rollkit query tx --type=hash $TX_HASH --chain-id wordle --output json | jq
 ```
 <!-- markdownlint-enable MD013 -->
 
 This should display an output like the following:
 
 ```json
```

Comment:
gh highlights it all in dark red, as it's not a valid json

Reply:
that's a gh UI issue. It looks good on the website.